<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <parent>
    <artifactId>catch-exception-parent</artifactId>
    <groupId>eu.codearte.catch-exception</groupId>
    <version>1.3.1-SNAPSHOT</version>
  </parent>
  <modelVersion>4.0.0</modelVersion>
  <packaging>pom</packaging>

  <artifactId>functional-tests</artifactId>

  <modules>
    <module>catch-exception-assertj17</module>
    <module>catch-throwable-assertj17</module>
    <module>catch-exception-assertj16</module>
    <module>catch-throwable-assertj16</module>
  </modules>

</project>